### ISSUE TITLE
About Prediction

### ISSUE THREAD

**Issue by bokgyu** [fabianbormann/Tensorflow_Segmentation#20]
I'm really sorry about it to make this issue with my problem

my question is this.
checkpoint_directory = "/checkpoints/"
img1 =np.float32(cv2.resize(cv2.imread("./test_data/tabby_cat.png"),(224,224)))
img1 =tf.placeholder(tf.expand_dims(img1, 0))

#img2 = tf.placeholder(tf.float32,shape=(1,224,224,3))
#img2 = {img2:img1}

checkpoint_file=tf.train.latest_checkpoint(checkpoint_directory)
session_conf = tf.ConfigProto(allow_soft_placement=True)
session_conf.gpu_options.allocator_type = 'BFC'
init = tf.initialize_all_variables()

trainset = open('data/stage_1_train_imgset/train.txt').readlines()
random_line = random.choice(trainset)
image_file = random_line.split(' ')[0]
ground_truth_file = random_line.split(' ')[1]
image = np.float32(cv2.imread('data' + image_file))
ground_truth = cv2.imread('data' + ground_truth_file[:-1], cv2.IMREAD_GRAYSCALE)

ground_truth = (ground_truth / 255) * 20

sess = tf.Session(config = session_conf)

with sess.as_default():
    
            
    #saver =tf.train.import_meta_graph("{}.meta".format(checkpoint_file))
    #saver.restore(sess,checkpoint_file)
    sess.run(init)
    deconv = DeconvNet()
    #feed_dict = {deconv.x:img2}
    
    #deconv.predict(image)
        
    tensors = [deconv.logits, deconv.prediction]
    C = sess.run(tensors,feed_dict ={deconv.x:img1})

I finished the "Training" with Deconv.py then tried to make prediction things.
Is there any wrong things for making prediction?
plz help me.

**Comment by fabianbormann** [comment]
[maybe] closed by e4d59e9
